app/models/queue_tabs/education_emo_completed_tasks_tab.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class EducationEMOCompletedTasksTab < QueueTab
```

Comment:
```suggestion
class EducationEmoCompletedTasksTab < QueueTab
```
@AimanK if we're following the same naming convention that we've done with the other EMO tabs, I believe that we only want to uppercase the "E" in EMO